### ISSUE TITLE
Explore and verify best placement of position information for slider interactions

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#280]
While testing #245 and #261, I found the flipped design was implemented as I requested.

That said, I want to note that when I use the slider interactions (individual left or right hands), I feel like I need position as part of the object response.

This is not the case with the Both Hands interaction where I receive comprehensive position information for each hand while interacting.

This is an interesting design issue. I am opening this issue for future investigation. It likely won't happen for the publication for this version.

At Tuesday (Dec 8th) design meeting, I also mentioned that I felt like I was missing position information sometimes. This issue can explore that "possible" design gap further as we move the current version of the sim towards publication.

**Comment by terracoda** [comment]
Oh, I addressed this issue in a quick meeting with @zepumph by opening #322. 
Learners will have access to position information in a dynamic description of the hands.

#322 addresses the concern I had, so I am closing this issue.
